GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -581,6 +581,7 @@ void Renderer::renderRenderData(RenderData* render_data,
                             break;
                         case Material::ShaderType::EXTERNAL_RENDERER_SHADER:
                             shader_manager->getExternalRendererShader()->render(
+                                    mv_matrix, glm::inverseTranspose(mv_matrix),
```

Comment:
Any benefits of running an inverse as an argument. Can't this go inside the render function?

You could save one inverse operation if you place this call after the null check for the material in render().